src/pages/home/report/ReportActionCompose.js

```diff
@@ -732,6 +938,28 @@ class ReportActionCompose extends React.Component {
                     <ExceededCommentLength commentLength={this.comment.length} />
                 </View>
                 {this.state.isDraggingOver && <ReportDropUI />}
+                {this.state.suggestedEmojis.length && this.state.shouldShowSuggestionMenu ? (
```

Comment:
Use `condition && <component />` rather than a ternary with `condition ? <component /> : null`

Reply:
sure, I'll change it, I just used the existing code as a style guide